my-app/src/Navbar/Badges.jsx

```diff
@@ -2,6 +2,59 @@ import React from "react";
 
 export default function Badges() {
     return (
+    <div>
         <h1>Badges </h1>
+        <a id="category-MASTER-trigger" href="#" class="badge-category-trigger">
+             
+              <img alt="Challenge Patches" class="badge-category-image" src="https://cdn.kastatic.org/images/badges/master-challenge-blue-medium.png"/>
+              <div aria-hidden="true" class="achievement-label">
+                Challenge Patches
+              </div>
+            </a>
+
+            <a id="category-DIAMOND-trigger" href="#" class="badge-category-trigger">
+              <img alt="Black Hole Badges" class="badge-category-image" src="https://cdn.kastatic.org/images/badges/eclipse-medium.png"/>
+              <div aria-hidden="true" class="achievement-label">
+                Black Hole Badges
+              </div>
+            </a>
+
+            <li class="badge-category platinum owned pure-u-1-3 pure-u-md-1-6" data-category="category-PLATINUM">
```

Comment:
My hunch is that you got this styling code from somewhere. Be sure to cite your sources!